src/gui_tool/gui/tinker_subuis/cashe_manager.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os.path as path
+
+class ListCasheManager(object):
```

Comment:
I think this is a typo? Should be Cache instead of Cashe

Reply:
lol Every time